cypress/fixtures/clientes.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Não vejo este _fixture_ sendo usada em lugar nenhum.

Se este for o caso, recomendo deletá-la.

Reply:
A propósito, esta não é a estrutura que o _frontend_ espera.

A estrutura correta é [esta](https://github.com/wlsf82/EngageSphere-Test-Design-Masterclass-Turma-3-new/blob/main/docs/Requirements.md#response-body).